formats/integers.ksy

```diff
@@ -1,58 +1,44 @@
 meta:
   id: integers
-  endian: le
 seq:
-  - id: magic1
-    contents: 'PACK-1'
-  - id: uint8
-    type: u1
-  - id: sint8
-    type: s1
-  - id: magic_uint
-    contents: 'PACK-U-DEF'
-  - id: uint16
-    type: u2
-  - id: uint32
-    type: u4
-  - id: uint64
-    type: u8
-  - id: magic_sint
-    contents: 'PACK-S-DEF'
-  - id: sint16
-    type: s2
-  - id: sint32
-    type: s4
-  - id: sint64
-    type: s8
-  - id: magic_uint_le
-    contents: 'PACK-U-LE'
-  - id: uint16le
-    type: u2le
-  - id: uint32le
-    type: u4le
-  - id: uint64le
-    type: u8le
-  - id: magic_sint_le
-    contents: 'PACK-S-LE'
-  - id: sint16le
-    type: s2le
-  - id: sint32le
-    type: s4le
-  - id: sint64le
-    type: s8le
-  - id: magic_uint_be
-    contents: 'PACK-U-BE'
-  - id: uint16be
-    type: u2be
-  - id: uint32be
-    type: u4be
-  - id: uint64be
-    type: u8be
-  - id: magic_sint_be
-    contents: 'PACK-S-BE'
-  - id: sint16be
-    type: s2be
-  - id: sint32be
-    type: s4be
-  - id: sint64be
-    type: s8be
+  - id: unsigned_min
+    type: unsigned
+  - id: unsigned_max
+    type: unsigned
+  - id: signed_min
+    type: signed
+  - id: signed_max
+    type: signed
```

Comment:
The idea of "integers" test is to test most bare minimum of functionality of KS — that is, bultin integer types. It makes sense to test default endianness here too (and this is why it originally had `endian: le`), but we need to keep it simple, i.e. no subtypes / substructures, no substreams, no variable length structures, etc.

I agree that it might make sense to remove `endian: le` and `contents: ...` from this KSY to make it even simpler, but I'm very much hesitating in adding more stuff (even if that's the stuff we normally take for granted), as it kind of breaks the idea of going from simple to more complicated in development of compilers/translators for new languages.

Reply:
I didn't know that was the idea behind the integers test. Makes sense to leave it as is.

How about a separate test like `integers_min_max` instead? Testing these values would properly fail on JS so I think it's a good idea. Might uncover something in other runtimes, too. I'll just go ahead and make a PR.

Reply:
Makes sense, thanks again for your contributions and effort!